padle/padle.py

```diff
@@ -105,8 +107,7 @@ async def get_today_guesses(self, user: discord.User, current_day: Union[str, in
             return None
         if user is None:
             return {}
-        async with self.config.user(user).all_guesses() as all_guesses:
-            return all_guesses.get(str(current_day))
+        return (await self.config.user(user).all_guesses()).get(str(current_day))
```

Comment:
So does this need a `with`? Usually, if it CAN take a `with`, it should, right? Or are you sure that this doesn't cause a resource leak?

Reply:
It doesn't need a with!  You only need a with if you need to mutate a mutable data type.  If you're just reading from it, you don't have to.